test/spec/modules/publinkIdSystem_spec.js

```diff
@@ -72,18 +72,33 @@ describe('PublinkIdSystem', () => {
       expect(result.callback).to.be.a('function');
     });
 
-    it('Use local copy', () => {
-      const result = publinkIdSubmodule.getId({}, undefined, TEST_COOKIE_VALUE);
-      expect(result).to.be.undefined;
-    });
-
     describe('callout for id', () => {
       let callbackSpy = sinon.spy();
 
       beforeEach(() => {
         callbackSpy.resetHistory();
       });
 
+      it('Has cached id', () => {
+        const config = {storage: {type: 'cookie'}, params: {}};
+        let submoduleCallback = publinkIdSubmodule.getId(config, undefined, TEST_COOKIE_VALUE).callback;
+        submoduleCallback(callbackSpy);
+
+        const request = server.requests[0];
+        const parsed = parseUrl(request.url);
+
+        expect(parsed.hostname).to.equal('proc.ad.cpe.dotomi.com');
+        expect(parsed.pathname).to.equal('/cvx/client/sync/publink');
```

Comment:
This test should have failed because it's still the old generate endpoint, right?

Reply:
yep, it did